pkgs/top-level/all-packages.nix

```diff
@@ -5919,6 +5919,8 @@ with pkgs;
 
   devpi-server = python3Packages.callPackage ../development/tools/devpi-server { };
 
+  dprint-plugins = recurseIntoAttrs (callPackage ../by-name/dp/dprint/plugins { });
```

Comment:
Can we realize to call it with `dprint.plugins` such as `ruby_3_4.gems.rubocop`? 

Or is the namespace way deprecated? It seems to frequently flatten multiple packages.

Reply:
changed it to `dprint.plugins`

Reply:
Well, I get issues when using it with treefmt-nix, looking at it.

Reply:
maybe I need to use `passthru.plugins`?